internal/log/log.go

```diff
@@ -0,0 +1,22 @@
+// Package log is a preflight-local logrus bridge, removing the
+// dependency on logrus' global logger instance.
+package log
```

Comment:
If we go with a log package now, we are kind of stuck with this forever right?

Reply:
Why would we be stuck with it forever?

Reply:
I missed that this is internal, so I guess we are free to change it at will and not affect the API layer.

Reply:
Yep. That's the idea. I expect it to change in the not-so-distant future, but the level of change to rearchitect this requires changing all of our check code in non-trivial ways. I considered this PR to be effectively a trivial change since the underlying behavior doesn't change.